examples/SAP-to-Database/Integrate SAP API to update database.md

```diff
@@ -0,0 +1,64 @@
+# Read SAP Inspection Master data and Update your mySQL database
```

Comment:
MySQL is the correct case. Run all descriptions through chatgpt to improve

Reply:
Resolved here [Improve example documentation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/23/commits/bf21323195d3826d88e16585d6051c519a9d8580)